attributes.rst

```diff
@@ -0,0 +1,146 @@
+.. index::
+    single: Attributes
+
+Symfony Attributes Overview
+===========================
+
+Attributes are the successor of annotation since PHP 8. Attributes are native
+to the language and Symfony takes full advantage of them across the framework
+ans its different components.
+
+This article lists supported attributes of the framework, as well as their
+usages in the documentation.
+
+Doctrine Bridge
+~~~~~~~~~~~~~~~
+
+* :doc:`UniqueEntity </reference/constraints/UniqueEntity>`
+
+Command
+~~~~~~~
+
+* :ref:`AsCommand <console_registering-the-command>`
+
+Contracts
+~~~~~~~~~
+
+* :ref:`Required <autowiring-calls>`
+* :ref:`SubscribedService <service-subscribers-service-subscriber-trait>`
+
+Dependency Injection
+~~~~~~~~~~~~~~~~~~~~
+
+* :ref:`AsTaggedItem <tags_as-tagged-item>`
+* :ref:`Autoconfigure <lazy-services_configuration>`
+* :ref:`AutoconfigureTag <di-instanceof>`
+* :ref:`TaggedIterator <tags_reference-tagged-services>`
+* :ref:`TaggedLocator <service-subscribers-locators_defining-service-locator>`
+* :ref:`Target <autowiring-multiple-implementations-same-type>`
+* :ref:`When <service-container_limiting-to-env>`
+
+EventDispatcher
+~~~~~~~~~~~~~~~
+
+* :ref:`AsEventListener <event-dispatcher_event-listener-attributes>`
+
+HttpKernel
+~~~~~~~~~~
+
+* :doc:`AsController </controller/service>`
+
+Messenger
+~~~~~~~~~
+
+* :ref:`AsMessageHandler <messenger-handler>`
+
+Routing
+~~~~~~~
+
+* :doc:`Route </routing>`
+
+Security
+~~~~~~~~
+
+* :class:`Symfony\\Component\\Security\\Core\\Validator\\Constraints\\UserPassword`
```

Comment:
Continuing my previous comment, I would also remove this as this is a validation constraint (which package provides something is an "implementation detail" for documentation readers)